### DISCUSSION TITLE
Workaround for JavaScript

### DISCUSSION THREAD

**Discussion by ghost** [Eloston/ungoogled-chromium#1645] [Q&A] [answered]
I have a YouTube and Gmail account, but I am unable to sign in due to JavaScript being disabled.
![Capture](https://user-images.githubusercontent.com/36653213/133873863-0572702b-c21c-469d-b1ee-7d2361a83f9c.PNG)
Is there a way to enable JavaScript temporarily, or perhaps install an extension that allows customized control of JavaScript like uMatrix does?

**Comment by networkException** [comment, marked as answer]
JavaScript is not disabled by default in any way, please check your settings and flags

**Reply by ghost** [reply]
You are correct, it's because I'm running uMatrix, thank you!
